app/views/dashboard/advisers/_advisers_table.html.erb

```diff
@@ -9,18 +9,29 @@
       </tr>
     </thead>
     <tbody>
-      <% advisers.each do |adviser| %>
-        <tr class="t-advisers-table-row">
-          <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
-          <td class="t-adviser-name"><%= adviser.name %></td>
-          <td>
-            <ul class="l-comma-separated-list">
-              <% adviser.qualifications.each do |qualification| %>
-                <li class="t-adviser-qualification"><%= qualification.name %></li>
-              <% end %>
-            </ul>
+      <% if firm.advisers.present? %>
+        <% firm.advisers.each do |adviser| %>
+          <tr class="t-advisers-table-row">
+            <td class="t-adviser-reference-number"><%= adviser.reference_number %></td>
+            <td class="t-adviser-name"><%= adviser.name %></td>
+            <td>
+              <ul class="l-comma-separated-list">
```

Comment:
Components are individual "things", while layouts style components for specific contexts. As this is just a list, it's a component rather than a 'layout of components'. Does that help?